src/Tribe/Utils/DST.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+
+class Tribe__Events__Utils__DST {
+
+	/**
+	 * @var bool|null
+	 */
+	protected $_in_dst;
+
+	/**
+	 * Tribe__Events__Utils__DST constructor.
+	 *
+	 * @param int|string $time    Either a UNIX timestamp or an english format date.
+	 * @param null|bool  $_in_dst An injectable DST status, meant for tests.
+	 */
+	public function __construct( $time, $_in_dst = null ) {
+		if ( ! is_numeric( $time ) ) {
+			$time = strtotime( $time );
+		}
+
+		$this->time    = $time;
```

Comment:
Every method in the class uses the `time` field but we only declare `_in_dst` - could we add `time` as a protected field, too?